core/service/izanami.cpp

```diff
@@ -77,11 +92,15 @@ namespace izanami {
         return false;
     }
 
+    bool InitializeEvent::isFinished() const {
+        return is_finished;
+    }
 
-    void InitializeEvent::clear() {
+    void InitializeEvent::finished() {
```

Comment:
OK. I will replace `InitializeEvent::finished()` to `InitializeEvent::finish()`.